src/shared/components/Studio/ExpandableStudioList.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { Collapse, Button, Empty } from 'antd';
+import { useHistory } from 'react-router-dom';
+
+import StudioListItem from './StudioListItem';
+import { StudioItem } from '../../views/StudioListView';
+
+import './ExpandableStudioList.less';
+
+const ExpandableStudioList: React.FC<{
+  studios: StudioItem[];
+  error: Error | null;
+}> = ({ studios, error }) => {
+  const history = useHistory();
+
+  const makeStudioUri = (studio: StudioItem) => {
```

Comment:
May be , this belongs to a utility file. With input parameter as a string, rather than studio object. This will make it less specific and re-usable.